### ISSUE TITLE
Change text on in-progress interviews page

### ISSUE THREAD

**Issue by ekressmiller** [mplp/docassemble-mlhframework#105]
We'd like to change the text on the My Interviews/ In progress forms page to also say "My Interviews"
![image](https://github.com/mplp/docassemble-mlhframework/assets/110936328/86562abb-ce2a-4d35-bf4c-84805e071865)

I tried changing PAGE_QUESTION in our [replacement interview_list.yml](https://github.com/mplp/docassemble-mlhframework/compare/main...MyInterviewsPage), but it seemed to have no effect. I also tried setting "page question" in the config as [suggested here](https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/9ef1c7df80905f1185333cbc3942e2a5bccc5095/docassemble/AssemblyLine/data/questions/interview_list.yml#L6), but again no apparent effect.

@BryceStevenWilley, do you have any thoughts?

**Comment by BryceStevenWilley** [comment]
I'm not sure what's happening here; I started up a brand new docassemble, only installed `mlhframework` on the `MyInterviewsPage` branch, and it shows "My Interviews" as the title.

I did change the configuration on `da-dev.mplp.org` but had some trouble getting it to show up. I had to re-start that "My Interviews page" from scratch, otherwise it still uses the same title it had loaded before. Going to https://da-dev.mplp.org/start/mlhframework/interview_list fixed it. That might fix your issue as well? Let me know!
